src/components/TodoFilter/TodoFilter.tsx

```diff
@@ -12,19 +1,66 @@
+import React from 'react';
+import { Filters } from '../../types/Filters';
 
-    <p className="control is-expanded has-icons-left has-icons-right">
-      <input
-        data-cy="searchInput"
-        type="text"
-        className="input"
-        placeholder="Search..."
-      />
-      <span className="icon is-left">
-        <i className="fas fa-magnifying-glass" />
-      </span>
+const filters = ['All', 'Active', 'Completed'];
 
-      <span className="icon is-right" style={{ pointerEvents: 'all' }}>
-        {/* eslint-disable-next-line jsx-a11y/control-has-associated-label */}
-        <button data-cy="clearSearchButton" type="button" className="delete" />
-      </span>
-    </p>
-  </form>
-);
+type Props = {
+  setSelectedFilter: React.Dispatch<React.SetStateAction<Filters>>;
+  setQuery: React.Dispatch<React.SetStateAction<string>>;
+  query: string;
+};
+
+export const TodoFilter: React.FC<Props> = ({
+  setSelectedFilter,
+  setQuery,
+  query,
+}) => {
+  const handleSelectFilter = (event: React.ChangeEvent<HTMLSelectElement>) => {
+    setSelectedFilter(event.target.value as Filters);
+  };
+
+  const handleInputChange = (event: React.ChangeEvent<HTMLInputElement>) => {
+    setQuery(event.target.value.trimStart());
+  };
```

Comment:
Your method names `handleSelectFilter` and `handleInputChange` are not following the naming conventions for methods in React. It's a good practice to name event handlers with the prefix 'on' and the part that describes what the event is. For example, `handleSelectFilter` can be renamed to `onFilterChange` and `handleInputChange` can be renamed to `onInputChange`. You can read more about it [here](https://medium.com/javascript-in-plain-english/handy-naming-conventions-for-event-handler-functions-props-in-react-fc1cbb791364).